pkg/services/featuremgmt/registry.go

```diff
@@ -129,7 +129,7 @@ var (
 			Description:  "Enables drag and drop for CSV and Excel files",
 			FrontendOnly: true,
 			Stage:        FeatureStageExperimental,
-			Owner:        grafanaDatavizSquad,
+			Owner:        grafanaFrontendPlatformSquad,
```

Comment:
Not sure this makes sense. Got balme points to Adela (DataViz), Zoltan (DataSources), Oscar Kilhed (dashboards-edit).
Shall we try Dashboards Edit?